### PR TITLE
unified namespace reference

### PR DESCRIPTION
**Pull request by zc2638**
Signed-off-by: zc <zc2638@qq.com>

**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:
> unified namespace reference

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: common/constants/default.go [hunk 1]
```diff
@@ -14,8 +14,8 @@ const (
 	// NodeName is for the clearer log of cloudcore.
 	NodeName = "NodeName"
 
-	KubeEdge          = "kubeedge"
-	KubeEdgeNameSpace = "kubeedge"
+	Name      = "kubeedge"
+	Namespace = Name
```
Comment (fisherxu) [inline common/constants/default.go:18]: `Namespace` and `Name` don’t sound so clearly when developers import...?

Comment (zc2638) [reply common/constants/default.go:18]: like `SystemNamespace` ?

Comment (zc2638) [reply common/constants/default.go:18]: For example k8s, defined as `NamespaceDefault`

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**